components/Badge/Badge.tsx

```diff
@@ -102,7 +102,7 @@ const StyledButtonBadge = styled('button', BADGE_BASE_STYLES, {
       right: 0,
       bottom: 0,
       left: 0,
-      borderRadius: '5px',
+      borderRadius: '$pill',
```

Comment:
```suggestion
      borderRadius: 'inherit',
```

Would avoid updating 2 values in case of a change. WDYT?